community/kernel/src/main/java/org/neo4j/kernel/impl/store/StoreFactory.java

```diff
@@ -621,6 +622,18 @@ private void createEmptyStore( File fileName, String typeAndVersionDescriptor, B
         }
     }
 
+    /**
+     * I.e. total number of used/unused records + 1
+     * @param storeFile {@link StoreFile} to get the name from.
+     * @param recordSize record size of that store.
+     * @return highId, i.e. an id one greater than the highest id in the store.
+     */
+    public long getHighId( StoreFile storeFile, int recordSize )
```

Comment:
Oh yeah, hmm you know what? We could instead just read the highId from `.id` file. I think that should work just fine instead.